metricflow/model/parsing/config_linter.py

```diff
@@ -80,17 +86,24 @@ def lint_file(self, file_path: str, file_name: str) -> List[ValidationIssue]:  #
                 if problem.rule == rules.key_duplicates.ID:
                     self.add_additional_key_duplicates_info(problem=problem)
 
-                level = ValidationIssueLevel.ERROR if problem.level == ERROR else ValidationIssueLevel.WARNING
-                issues.append(
-                    ValidationIssue(
-                        level=level,
-                        context=ValidationContext(file_name=file_name, line_number=problem.line),
-                        message=problem.desc,  # type: ignore[misc]
+                if problem.level == ERROR:
+                    issues.append(
+                        ValidationError(
+                            context=FileContext(file_name=file_name, line_number=problem.line),
+                            message=problem.desc,  # type: ignore[misc]
```

Comment:
Context on why `# type: ignore[misc]`?

Reply:
The [YamlLint package isn't currently typed](https://github.com/adrienverge/yamllint/blob/40cab7f999b7d8f4f143f22f6932126274238e0d/yamllint/linter.py#L33) 🙃 I've debated about spending some time contributing to the package to make it typed, but time is a lacking. So `problem.desc` has a type of `Any`. I could either do an `assert isinstance(problem.desc, str)`, try to stringify it `str(probelm.desc)`, or just ignore it because we know it's always a string. In this instance because `problem.desc` is actually always a string, I decided to just ignore the type problem.